04/app.js

```diff
@@ -84,6 +86,11 @@ class App extends React.Component {
             users: currUsers,
         });
     }
+
+    filterHandler = e => {
+        const { searchQuery } = this.state
+        this.setState({ searchQuery: e.target.value.toLowerCase() })
```

Comment:
aaa ok, tutaj to robisz - może zostać, ale z drugiej strony wyżej mamy w jednym miejscu i użytkownik nie jest zdziwiony dlaczego jego wielka litera stała się mała ;P